sources/platform/console/index.md

```diff
@@ -10,17 +10,65 @@ slug: /console
+
+:::info CAPTCHA
+We are using Google reCaptcha to prevent spam accounts. Usually you will not see it, but if Google evaluates your browser as suspicious, they will ask you to solve a reCaptcha before we create your account and send you the verification email.
+:::
+
+If you did not receive the email, you can go to the [sign in page](https://console.apify.com/sign-in). There you will either proceed to our verification page right away, or you can sign in and you will be redirect after. On the verification page you can click on the **Resend verification email** button to send the email again.
+
+![Apify console email verification page](./images/console-email-verification-page.png)
+
+### Google or GitHub
+
+If you do not want to create a new password for Apify Console, you can also use your Google or GitHub account to sign up. To do that, click on the **Sign up with Google** or **Sign up with GitHub** buttons. You will proceed to the corresponding authentication page where you will need to sign in and authorize Apify to access your account. Then when you come back to the Apify Console we will create your account.
+
+If you used Google to sign up, there is no verification step after this and you can start to use the Apify Console right away.
+On the other hand if you used GitHub to create your account and your github account does not have verified email, we will ask you to verify your email address before you can start using Apify Console. The process works the same as the process for email and password authentication.
+
+If you already have an account in Apify Console connected to Google or GitHub, clicking on these buttons on the sign up page will not create a new account and instead it will directly sign you in.
+
+## Sign-in
+
+To sign in to your account please go to the [sign in page](https://console.apify.com/sign-in).
+There you can use any of the authentication methods you have set up for your account. If you have multiple authentication methods, you can choose which one you want to use by using on the corresponding button or form.
+
+If you sign in through email and password and have two-factor authentication enabled on your account, you will proceed to the two-factor authentication page after you enter your password. There you will need to enter the code from your authenticator app to sign in.
+If you do not have two-factor authentication enabled, you will be directly redirected to the Apify Console.
+
+![Apify console sign in form](./images/console-sign-in-form.png)
+
+### Forgotten password
+
+In case you forgot your password, you can click on the **Forgot your password?** link which will redirect you to the [forgot password page](https://console.apify.com/forgot-password). There you will need to enter your email address and click on the **Reset password** button. We will then send an email to the address connected to your account with a link to the password reset page which will allow you to change your password.
```

Comment:
```suggestion
In case you forgot your password, you can click on the **Forgot your password?** link, which will redirect you to the [forgot password page](https://console.apify.com/forgot-password). There, you will need to enter your email address and click on the **Reset password** button. We will then send an email to the address connected to your account with a link to the password reset page, which will allow you to change your password.
```